doc/source/conf.py

```diff
@@ -26,12 +26,33 @@
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['IPython.sphinxext.ipython_console_highlighting',
-              'IPython.sphinxext.ipython_directive']
+              'IPython.sphinxext.ipython_directive',
+              'sphinx_gallery.gen_gallery',
+              'sphinx.ext.autosummary', 'sphinx.ext.intersphinx',
+]
+
+def setup(app):
+    app.add_stylesheet('custom.css')  # may also be an URL
 
 # Add any paths that contain templates here, relative to this directory.
 
 templates_path = ['_templates']
 
+autosummary_generate = True
+
+# Sphinx gallery configuration
+sphinx_gallery_conf = {
+    'examples_dirs': ['../../gallery'],
```

Comment:
I just noticed that the `nyc_boros` file is plotting/saving files that are used later in the documentation. There's a note in there about auto-building as a part of the documentation. I can sphinx-gallerify this example and then you wouldn't need the `savefigs` in there anymore...you could just use the images that sphinx-gallery generates. LMK.